python/catkin/builder.py

```diff
@@ -287,17 +288,8 @@ def get_multiarch():
 
 def get_python_install_dir():
     # this function returns the same value as the CMake variable PYTHON_INSTALL_DIR from catkin/cmake/python.cmake
```

Comment:
This function is used here:

https://github.com/ros/catkin/blob/kinetic-devel/python/catkin/builder.py#L602

And that is used to generate a setup file here:

https://github.com/ros/catkin/blob/kinetic-devel/python/catkin/builder.py#L641

So maybe it only affects `catkin_make_isolated` in non-merged spaces, but I still think it could result in the `PYTHONPATH` pointing to one place and the code getting installed to another.